products/ansible-core.md

```diff
@@ -0,0 +1,57 @@
+---
+permalink: /ansible-core
+layout: post
+title: Ansible-core
+versionCommand: ansible --version
+releasePolicyLink: https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html
+changelogTemplate: https://github.com/ansible/ansible/blob/stable-__RELEASE_CYCLE__/changelogs/CHANGELOG-v__RELEASE_CYCLE__.rst
+releaseDateColumn: true
+sortReleasesBy: "releaseDate"
+activeSupportColumn: false
+eolColumn: Supported
+iconSlug: ansible
+auto:
+  git: https://github.com/ansible/ansible.git
+releases:
+  - releaseCycle: "2.12"
+    releaseDate: 2021-11-05
+    eol: false
+    latest: "2.12.9"
+    latestReleaseDate: 2022-09-12
+
+  - releaseCycle: "2.11"
+    releaseDate: 2021-04-26
+    eol: false
+    latest: "2.11.12"
+    latestReleaseDate: 2022-05-23
+    
+  - releaseCycle: "2.10"
+    releaseDate: 2020-08-13
+    eol: false
+    latest: "2.10.17"
+    latestReleaseDate: 2022-01-31
+
+  - releaseCycle: "2.9"
+    releaseDate: 2019-10-31
+    eol: false
```

Comment:
```suggestion
    eol: true
```

Reply:
@felixfontein could you also mention their eol dates because we prefer to use dates instead of true [ if we got the related dates]